src/main.py

```diff
@@ -1,8 +1,22 @@
 # Resolve the problem!!
 
+import re
+
 
 def run():
-    # Start coding here
+    text = ""
+    letters = "[a-z]"
+    message = ""
+
+    with open("encoded.txt", mode="r", encoding="utf-8") as f:
+        for line in f:
+            text += line
+    
+    for letter in text:
+        if re.match(letters, letter):
+            message += letter
```

Comment:
Puedes usar otros métodos de `re` para encontrar todas las coincidencias, estos son más rápidos que un ciclo for.

Reply:
https://docs.python.org/3/library/re.html

Reply:
Hecho. El código quedó mucho más legible que antes. :D